app/domain/people_skills_filter.rb

```diff
@@ -23,4 +28,8 @@ def filter_by_rated
     end
     entries
   end
+
+  def filter_by_level
+    entries.where('level >= ?', level)
+  end
```

Comment:
Here you can just use
`return entries if level.nil?`

The idea of the scope method is to just filter everything using the parameters passed on initialization. If you do an if else statement, it won't run all the filters. It's not the scope methods concern whether or not there even is a level value. It just want's to run all filters over it. When the filter does not have the required value, it just returns the received entries